test/src/locator.spec.ts

```diff
@@ -581,6 +581,38 @@ describe('Locator', function () {
     });
   });
 
+  describe('Locator.last', () => {
+    it('should time out when any one of the locaters do not match', async () => {
```

Comment:
could you please add some tests for the following scenarios?

1) Locator.last waits for multiple elements that a clickable and it succeeds clicking only on the last one
2) Locator.last waits for multiple elements but only the last one is clickable and it succeeds clicking on the last one